### DISCUSSION TITLE
Navigation to personal Projects

### DISCUSSION THREAD

**Discussion by charleskawczynski** [community/community#45040] [Projects and Issues] [answered]
I don't see anywhere that I can access Projects on the GH mobile app. Is this possible?

**Comment by jge162** [comment, marked as answer]
@charleskawczynski It is currently not possible to access Projects on the GitHub mobile app. The app primarily focuses on allowing users to view and interact with their code and repositories, while Projects can only be accessed and created on the GitHub website.
